app/plex_auto_collections.py

```diff
@@ -17,187 +17,255 @@
-        methods = [m for m in collections[c] if m not in ("details", "subfilters")]
+        print("| \n|===================================================================================================|\n|")
+        print("| Updating collection: {}...".format(c))
+        tmdbID = None
+        methods = [m for m in collections[c] if m not in ("details", "subfilters", "file")]
         subfilters = []
         if "subfilters" in collections[c]:
             for sf in collections[c]["subfilters"]:
                 sf_string = sf, collections[c]["subfilters"][sf]
                 subfilters.append(sf_string)
         for m in methods:
-            if isinstance(collections[c][m], list):
-                # Support multiple imdb/tmdb/trakt lists
-                values = collections[c][m]
+            if collections[c][m]:
+                values = collections[c][m] if isinstance(collections[c][m], list) else str(collections[c][m]).split(", ")   # Support multiple imdb/tmdb/trakt lists
+                for v in values:
+                    m_print = m[:-1] if m[-1:] == "s" else m
+                    print("| Processing {}: {}".format(m_print, v))
+                    if m == "actors" or m == "actor":       v = get_actor_rkey(plex, v)
+                    met = m
+                    if m == "tmdbID":
+                        met = "tmdb-list"
+                        if not tmdbID:      tmdbID = v
+                        v = "https://www.themoviedb.org/collection/" + v
+
+                    check = True
+                    if (m == "tmdbID" or m == "tmdb-list") and not TMDB.valid:
+                            print("| Config Error: {} skipped. tmdb incorrectly configured".format(m))
+                            check = False
+
+                    if check:
+                        try:                            missing = add_to_collection(config_path, plex, met, v, c, subfilters)
+                        except UnboundLocalError:       missing = add_to_collection(config_path, plex, met, v, c)               # No sub-filters
+                        except (KeyError, ValueError) as e:
+                            print(e)
+                            missing = False
+                        if missing:
+                            if libtype == "movie":
+                                method_name = "IMDb" if "imdb" in m else "Trakt" if "trakt" in m else "TMDb"
+                                print("| {} missing movies from {} List: {}".format(len(missing), method_name, v))
+                                if Radarr.valid:
+                                    radarr = Radarr(config_path)
+                                    if radarr.add_movie:
+                                        print("| Adding missing movies to Radarr")
+                                        add_to_radarr(config_path, missing)
+                                    elif not headless and input("| Add missing movies to Radarr? (y/n): ").upper() == "Y":
```

Comment:
I was shooting for three states here (true, false, and unset) where the user is only prompted if it's unset. This looks like it'll prompt the use if `radarr.add_movie` is false or unset?

Reply:
Ahh i see now thats an easy fix